torch/_dynamo/output_graph.py

```diff
@@ -2229,6 +2198,274 @@ def maybe_lift_tracked_freevar_to_input(self, arg):
             return arg
         return self.lift_tracked_freevar_to_input(arg)
 
+    # See NOTE: [Auto lift basic free symbols when create_graph_input] for overall design
+    # This function should be used to track the unbacked symbols with its proxy created during
+    # dynamo tracing. It's invoked in wrap_fx_proxy for every proxied call for tensors and symbols.
+    # So that subgraph knows how to bind a symbol input with parent's proxy.
+    def lazy_bind_unbacked_symbols(self, example_value, e_proxy: torch.fx.Proxy):
```

Comment:
This doesn't completely tell me what this function does, or why it's lazy.

Is "This function finds all unbacked symbols in example_value and makes them inputs to the current subgraph tracer" the right description?

Reply:
`This function finds all unbacked symbols in example_value` , yes.  No for `makes them inputs to the current subgraph tracer`. It just records the mapping of symbol to proxy in current tracer. When subgraph tries to find the proxy it will create the proxy using the callback in current tracer, child tracer will bind the newly created proxy to its input.